doc/usage/extensions/inheritance.rst

```diff
@@ -97,6 +97,10 @@ It adds this directive:
    .. versionchanged:: 1.7
       Added ``top-classes`` option to limit the scope of inheritance graphs.
 
+   One of the predefined style can be selected with the directive ``style``.
+   It supports two different values: ``default``, which generates a diagram
+   with normal arrows; and ``uml`` which generates UML-like arrows.
+
```

Comment:
Please add `versionchanged` tag here? I think this would be added into 3.3.0 release.